src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionUrgencyEnum.java

```diff
@@ -9,15 +9,17 @@
 @AllArgsConstructor
 @Getter
 public enum TranscriptionUrgencyEnum {
-    STANDARD(1),
-    OVERNIGHT(2),
-    OTHER(3),
-    WORKING_DAYS_3(4),
-    WORKING_DAYS_7(5),
-    WORKING_DAYS_12(6),
-    WORKING_DAYS_2(7);
+    STANDARD(1, 999, "Standard"),
```

Comment:
we seem to be replicating the data in the database here. i'm not sure thats a good plan. would that make the data in the database redundant?

Reply:
Yeah good question @cmagowan-hmcts. @mestebanez any specific reason behind this change?

Reply:
yep assumed this enum was specifically for test purposes. I am changing to read urgency details from the database